apps/cli/src/node/docker-compose-explorer.yaml

```diff
@@ -5,22 +5,19 @@ services:
         image: postgres:15-alpine
         command: ["createdb", "squid"]
         environment:
-            PGHOST: database
-            PGPORT: 5432
-            PGUSER: postgres
-            PGPASSWORD: password
-            PGDATABASE: postgres
-        depends_on:
-            database:
-                condition: service_healthy
+            PGHOST: ${SUNODO_DB_HOST:-database}
```

Comment:
I'd use the same env var names of the standard postgres

Reply:
You mean the [libpq ones](https://www.postgresql.org/docs/current/libpq-envars.html) (`PGUSER`, `PGPASS`, ...) or the [Official Postgres Docker Image ones.](https://hub.docker.com/_/postgres) (`POSTGRES_USER`, `POSTGRES_PASSWORD`, ...)?

Reply:
I decided to go with `libpq` ones. See fixup [2f9f6f2](https://github.com/sunodo/sunodo/pull/344/commits/2f9f6f20d3c01ebd0d7e0015fe41ab4fba58e029)

Reply:
Yeah, the libpq ones